src/main/java/frc/robot/maps/TempestMap.java

```diff
@@ -1,32 +1,114 @@
 package frc.robot.maps;
 
+import java.util.function.BooleanSupplier;
+
 import com.chopshop166.chopshoplib.RobotMapFor;
 import com.chopshop166.chopshoplib.maps.DifferentialDriveMap;
 import com.chopshop166.chopshoplib.outputs.EncodedSpeedController;
 import com.chopshop166.chopshoplib.outputs.SendableSpeedController;
 import com.chopshop166.chopshoplib.sensors.MockEncoder;
 import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
 
+import edu.wpi.first.wpilibj.AnalogInput;
+import edu.wpi.first.wpilibj.AnalogTrigger;
+import edu.wpi.first.wpilibj.SpeedControllerGroup;
+import edu.wpi.first.wpilibj.Talon;
+import edu.wpi.first.wpilibj.Victor;
+
 @RobotMapFor("Tempest")
 public class TempestMap extends RobotMap {
 
     @Override
     public DifferentialDriveMap getDriveMap() {
         return new DifferentialDriveMap() {
+            @Override
+            public EncodedSpeedController getLeft() {
+
+                SendableSpeedController motors =  SendableSpeedController.wrap(new SpeedControllerGroup(new WPI_TalonSRX(4), new WPI_TalonSRX(1)));
+                return EncodedSpeedController.join(motors,new MockEncoder());
+
+
+            }
 
             @Override
             public EncodedSpeedController getRight() {
-                final SendableSpeedController rightGroup = SendableSpeedController.group(new WPI_TalonSRX(2),
-                        new WPI_TalonSRX(3));
-                return EncodedSpeedController.join(rightGroup, new MockEncoder());
+                SendableSpeedController motors = SendableSpeedController
```

Comment:
Instead of calling `SendableSpeedController.wrap(new SpeedControllerGroup(x, y, z))`, call `SendableSpeedController.group(x, y, z)`.